pmm-app/src/pmm-qan/panel/components/Details/Details.service.ts

```diff
@@ -14,7 +14,7 @@ const getExample = async ({
     tables,
   };
 
-  return apiRequestQAN.post<QueryExampleResponse, any>('/ObjectDetails/GetQueryExample', data);
+  return apiRequestQAN.post<QueryExampleResponse, any>('/v1/qan/query:getExample', data);
```

Comment:
I think the version shouldn't be specified here, but in the `components/helpers/api.ts`. Specifying it here results in:

```
http://localhost/v0/qan/v1/qan/metrics:getFilters
```

Reply:
Updated.